packages/flutter/lib/src/material/chip_theme.dart

```diff
@@ -326,6 +328,19 @@ class ChipThemeData extends Diagnosticable {
   ///  * [shadowColor]
   final Color selectedShadowColor;
 
+  /// Whether or not to show a check mark when [selected] is true.
+  ///
+  /// For instance, the [ChoiceChip] sets this to false so that it can be
+  /// be selected without showing the check mark.
+  ///
+  /// Defaults to true.
+  final bool showCheckmark;
```

Comment:
I don't think you should say `defaults to true` here since in the context of the ThemeData it defaults to null. I'd just remove the line

Reply:
keeping for file consistency